waveshare213v2/doc.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+// Package epd controls Waveshare e-paper series displays.
+//
+// More details
+//
+// Datasheets
+//
+// https://www.waveshare.com/w/upload/d/d5/2.13inch_e-Paper_Specification.pdf
+//
+// Product page:
+//
+// 2.13 Inch version 2: https://www.waveshare.com/wiki/2.13inch_e-Paper_HAT
+
```

Comment:
documentation won't show up if you keep an empty line.
You can confirm locally with

go get golang.org/x/tools/cmd/godoc
godoc -http=:6060

then visit http://localhost:6060/pkg/periph.io/x/devices/v3/epd/